easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-intel-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'nose-parameterized'
+version = '0.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'hmat://github.com/wolever/nose-parameterized'
+description = """Parameterized testing with any Python test framework."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('Python', '3.5.2')]
+
+options = {'modulename': 'nose_parameterized'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/nose_parameterized-%(version)s-py%(pyshortver)s.egg'],
```

Comment:
@wpoely86 we usually only check for `lib/python%(pyshortver)s/site-packages`, since the `.egg` can be either a file or a dir, depending on the situation...